src/activation.rs

```diff
@@ -52,6 +52,16 @@ impl Project {
                     }),
             ),
             (String::from("PIXI_IN_SHELL"), String::from("1")),
+            // Keep INIT_CWD or set to current working directory.
+            (
+                String::from("INIT_CWD"),
+                std::env::var("INIT_CWD").unwrap_or_else(|_| {
```

Comment:
maybe we could extract this in separate function like we do it for `home_path` or `cached_dir` and store it `pixi_config`

Reply:
Moved it to the get_task_env function. Where it is the only place it's used.